.github/workflows/pull-request.yml

```diff
@@ -21,7 +21,8 @@ jobs:
         env:
           packageUser: ${{ github.actor }}
           packagePAT: ${{ secrets.GITHUB_TOKEN }}
-        run: ./gradlew build
+        run: |
+          ./gradlew build
```

Comment:
do we need this change? looks like it is effectively the same

Reply:
Changed